lib/perf_hooks.js

```diff
@@ -145,6 +146,13 @@ function now() {
   return hr[0] * 1000 + hr[1] / 1e6;
 }
 
+function getMilestoneTimestamp(milestoneIdx) {
+  const us = milestones[milestoneIdx];
```

Comment:
`us`? I think you want `ns` for nanoseconds, right?

(If I’m wrong: Can you spell ‘microseconds’ out? We can’t use µ in our source code because it’s not ASCII, but it might not be obvious to everybody that it’s the “American” spelling of µs.)

Reply:
You're right, this is counted in nanoseconds.